src/pages/graphql/schema/customer/mutations/confirm-email.md

```diff
@@ -0,0 +1,64 @@
+---
+title: confirmEmail mutation | Commerce Web APIs
+---
+
+# changeCustomerPassword mutation
+
+Use the `confirmEmail` mutation to confirm the email address of a customer.
+
+## Syntax
+
+`mutation: {confirmEmail(input: ConfirmEmailInput!) {CustomerOutput}}`
+
```

Comment:
Add the following lines below the syntax. I will have to go back and a link to the reference once we've generated it.

```
## Reference

The `confirmEmail` reference provides detailed information about the types and fields defined in this query.
```

Reply:
Also add the following heading before the example

```
## Example usage
```